server/infrastructure/portal/mapper/PortalDataMapper.ts

```
@@ -0,0 +1,150 @@
+// server/infrastructure/portal/mapper/PortalDataMapper.ts
+import { CourseAreaType, EvaluationType } from '@/server/domain/course-offering/models/CourseOffering';
```

ESLint

[error] Imports "CourseAreaType" are only used as type.

(@typescript-eslint/consistent-type-imports)